book/src/super-sql/operators/count.md

```diff
@@ -0,0 +1,87 @@
+### Operator
+
+[✅](../intro.md#data-order)&emsp; **count** &mdash; emit a running count of input values
+
+>[!TIP]
+> For final counts across all input values, see the [count](../aggregates/count.md) aggregate function.
+
+### Synopsis
+
+```
+count [ <record-expr> ]
+```
+### Description
+
+The `count` operator produces a running count of input values as a field in
+an output record, including some or all of parts of input values in the output.
+
+If the optional `<record-expr>` is absent, the output record contains the
+input value in an element of [derived field name](../types/record.md#derived-field-names)
+`that` and a field of name `count` that contains the numeric count.
```

Comment:
Nit: I think a "show, don't tell" approach makes this clearer and more concise. I'd also move it after the next paragraph.

```suggestion
Without `<record-expr>`, `count` is equivalent to `count {that:this,count}`.
```